core/services/checkable.go

```diff
@@ -1,39 +1,11 @@
 package services
 
 import (
-	"errors"
-	"testing"
+	"github.com/smartcontractkit/chainlink-relay/pkg/services"
 )
 
-// Checkable should be implemented by any type requiring health checks.
-// From the k8s docs:
-// > ready means it’s initialized and healthy means that it can accept traffic in kubernetes
-// See: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
-type Checkable interface {
-	// Ready should return nil if ready, or an error message otherwise.
-	Ready() error
-	// HealthReport returns a full health report of the callee including it's dependencies.
-	// key is the dep name, value is nil if healthy, or error message otherwise.
-	// See CopyHealth.
-	HealthReport() map[string]error
-	// Name returns the fully qualified name of the component. Usually the logger name.
-	Name() string
-}
+// Deprecated: use services.HealthReporter
```

Comment:
nit: i understand the intention of this comment, but it's confused because this package is also `services`. how about listing the fully qualified relayer package name in the comment?

Reply:
Hmmm but this generates a godoc link since we are actually importing it as services. Should we alias it?